csharp/src/ProtocolBuffers/Descriptors/FileDescriptor.cs

```diff
@@ -54,6 +54,22 @@ public sealed class FileDescriptor : IDescriptor<FileDescriptorProto>
         private readonly IList<FileDescriptor> publicDependencies;
         private readonly DescriptorPool pool;
 
+        public enum Syntax
+        {
+            UNKNOWN,

```

Comment:
Why have the UNKNOWN here? This isn't a proto enum, so I don't think we really need it...